DataCleaning/splits_dividend_from_api.ipynb

```diff
@@ -0,0 +1,8756 @@
+      "Character not found\n",
+      "PYR found! Appending stats\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "RAS found! Appending stats\n",
+      "REE found! Appending stats\n",
+      "Character not found\n",
+      "RML found! Appending stats\n",
+      "RPM found! Appending stats\n",
+      "Character not found\n",
+      "Character not found\n",
+      "SHE found! Appending stats\n",
+      "SHG found! Appending stats\n",
+      "Character not found\n",
+      "SHP found! Appending stats\n",
+      "SKY found! Appending stats\n",
+      "SLA found! Appending stats\n",
+      "Character not found\n",
+      "SOV found! Appending stats\n",
+      "SPA found! Appending stats\n",
+      "SPN found! Appending stats\n",
+      "Character not found\n",
+      "SRL found! Appending stats\n",
+      "Character not found\n",
+      "STK found! Appending stats\n",
+      "Character not found\n",
+      "SWP found! Appending stats\n",
+      "Character not found\n",
+      "TBA found! Appending stats\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "TMG found! Appending stats\n",
+      "TMH found! Appending stats\n",
+      "TMS found! Appending stats\n",
+      "TOR found! Appending stats\n",
+      "Character not found\n",
+      "TPG found! Appending stats\n",
+      "Character not found\n",
+      "TRN found! Appending stats\n",
+      "TRU found! Appending stats\n",
+      "TSI found! Appending stats\n",
+      "Character not found\n",
+      "TUA found! Appending stats\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "UNI found! Appending stats\n",
+      "Character not found\n",
+      "Character not found\n",
+      "VOR found! Appending stats\n",
+      "VR1 found! Appending stats\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "WCG found! Appending stats\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "WSR found! Appending stats\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "Character not found\n",
+      "ZEO found! Appending stats\n",
+      "Character not found\n",
+      "Character not found\n",
+      "COF found! Appending stats\n",
+      "DBI found! Appending stats\n",
+      "ECF found! Appending stats\n",
+      "GDC found! Appending stats\n",
+      "Character not found\n",
+      "HMC found! Appending stats\n",
+      "Character not found\n",
+      "LFG found! Appending stats\n",
+      "MGF found! Append
```

Comment:
Do you know if the dividends is in $ or we need a formula to convert them to dollar? We need to know this.